napari/layers/vectors/_vector_utils.py

```diff
@@ -185,8 +185,10 @@ def fix_data_vectors(
     if ndim is not None and ndim != data_ndim:
         raise ValueError(
             trans._(
-                "Vectors dimensions must be equal to ndim",
+                f"Vectors dimensions ({data_ndim}) must be equal to ndim ({ndim})",
```

Comment:
[As the docs recommend](https://napari.org/developers/translations.html?highlight=translation#additional-arguments-inside-strings), likely because word order is language dependent.

```suggestion
                "Vectors dimensions ({data_ndim}) must be equal to ndim ({ndim})",
```

Reply:
New PyCharm feature with auto adding `f` on begin of string